include/constants.h

```diff
@@ -13,6 +15,14 @@ namespace reinforcement_learning {  namespace name {
       const char *const  INITIAL_EPSILON         = "initial_exploration.epsilon";
       const char *const  LEARNING_MODE           = "rank.learning.mode";
       const char* const  PROTOCOL_VERSION             = "protocol.version";
+      const char* const  HTTP_API_KEY            = "http.api.key";
+      const char *const  INTERACTION             = "interaction";
+      const char *const  OBSERVATION             = "observation";
+      const char *const  CONFIG_SECTION          = "config.section";
+      const char *const  EH_HOST                 = ".eventhub.host";
```

Comment:
I don't understand the change here? I was under the impression the idea of `reinforcement_learning::name` was the each constant mapped 1:1 with keys that could exist in the configuration JSON? Can you explain the intention here?

Reply:
Well, I needed these constants. Moved it to different namespace for readability.